src/routes/Home/index.tsx

```diff
@@ -1,15 +1,30 @@
 import { useTranslation } from 'react-i18next';
-import { Counter } from '@/features';
 import { Title } from '@/components';
+import { ColorCard } from '@jod/design-system';
 
 const Home = () => {
   const { t } = useTranslation();
 
   return (
     <div>
       <Title value={t('home')} />
-      <h1>{t('home')}</h1>
```

Comment:
`h1` removed as the biggest **ColorCard** has `h1` and page should have only one `h1` tag at all times at the same time.